daeun/2차원으로-만들기.js

```diff
@@ -0,0 +1,7 @@
+function solution(num_list, n) {
+    let answer = [];
+    while (num_list.length) {
+        answer.push(num_list.splice(0, n))
+    }
+    return answer;
+}
```

Comment:
```suggestion
function solution(num_list, n) {
    const numList = [...num_list];
    let answer = [];
    
    while (numList.length) {
        answer.push(numList.splice(0, n))
    }
    
    return answer;
}
```
- `while`문을 이용한 방법 잘봤습니다 👍
- 만약 인자로 넘겨받은 num_list 라는 배열을 이용 시 splice 하게되면 함수 밖 다른 곳에서 num_list를 이용하고 있었다면 오류가 발생했을지도 모릅니다..!
- 이때는 num_list를 깊은복사하여 사용하는 방법이 있습니다.